nilearn/utils/tests/test_discovery.py

```diff
@@ -39,7 +39,7 @@ def test_all_functions(
     """Check number of functions in public API."""
     fn = all_functions()
     print(fn)
-    assert len(fn) == 170
```

Comment:
@Remi-Gau Are you sure about this change. `test_discovery.test_all_function` is failing on main at this line.